Docs/source/usage/parameters.rst

```diff
@@ -2724,6 +2724,8 @@ Lookup tables store pre-computed values for functions used by the QED modules.
 
         * ``qed_bw.save_table_in`` (`string`): where to save the lookup table
 
+      Alternatively, the lookup table can be generated using a standalone tool (see workflows section).
```

Comment:
Can we add a link to the specific workflow sub-section?

Reply:
should be ok now